### PR TITLE
Decompiled .rsrc files to get .rdef files

### PR DESCRIPTION
**Pull request by ghost**
I decompiled both .rsrc files. I was told to do this on one GCI task, so I thought it couldn't hurt doing the same here as well (;

### CODE REVIEW

**Comment by waddlesplash** [comment]
You're supposed to create a recipe, right? Shouldn't you make a [Make|Jam]file? Please submit these as one pull request.

**Comment by ghost** [comment]
Alright. I'll to that then (;

**Comment by ghost** [comment]
I am not sure what happened, but I guess I'll undo this commit for now. When making a Makefile I realised that it's good you didn't pull the changes.
I decompiled the .rsrc files, but I am not able to compile them again due to rc reporting "syntax error, unexpected TYPECODE". I am not sure what it means, but i don't understand why this happened either. After all, I didn't change the file myself.

File: src/TimeCop_daemon.rdef [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+	variety = B_APPV_DEVELOPMENT,
+	internal = 0,
+
+	short_info = #'CSTR' array {
+
+	},
+	long_info = #'CSTR' array {
+
+	}
+};
```
Comment (scottmc) [inline src/TimeCop_daemon.rdef:70]: For short_info and long_info you should remove the #'CSTR' array { } part and replace it with something like:

short_info="Short description"

and

long_info="The text here might be the same as your summary entry in the recipe file"

Comment (ghost) [inline src/TimeCop_daemon.rdef:70]: I see. Thank you (;

**Comment by ghost** [comment]
Now the .rdef files are compiling. I created a Makefile, added a include and changed a define respectively.

**Comment by waddlesplash** [comment]
Thanks!

**Pull request merged**